tests/fixtures/apidoc/content_export_info.json

```diff
@@ -0,0 +1 @@
+katello.json
```

Comment:
I guess this is what Github looks like when you symlink a file.. :shrug:

Reply:
Yeah, the red is annoying but the link is correct.